packages/node_modules/sample-browser-auth-implicit/app.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env browser */
+/* global ciscospark */
+/* eslint-disable camelcase */
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable no-alert */
+/* eslint-disable no-console */
+/* eslint-disable require-jsdoc */
+
+// Declare some globals that we'll need throughout
+const spark = ciscospark.init();
+
+spark.once(`ready`, () => {
+
+  // Now, let's set up the event listener for the Authenticate Button
+  document.getElementById(`authorization`).addEventListener(`submit`, (event) => {
+    // let's make sure we don't reload the page when we submit the form
+    event.preventDefault();
+
+    // initiate the login sequence if not authenticated
+    spark.authorization.initiateLogin();
+  });
+
+  // Now, let's set up the event listener for the Authenticate Button
+  document.getElementById(`logout`).addEventListener(`submit`, (event) => {
+    // let's make sure we don't reload the page when we submit the form
+    event.preventDefault();
+
+    if (spark.canAuthorize) {
```

Comment:
maybe add a console statement or something so that the button doesn't seem broken when the user isn't yet logged in.

Reply:
would an alert that asks the user to login in work?

Reply:
console.log should be enough